roles/etcd/tasks/rpm.yml

```diff
@@ -67,6 +59,9 @@
   notify:
   - restart etcd
 
+- name: Ensure etcd owns the files in it's config dir
+  command: "chown etcd:etcd {{ etcd_conf_dir }}/ -R"
```

Comment:
Reason for switching to `chown`?

Reply:
etcd failed to start without this.  It didn't have permissions to read the certs which were all owned by root and mod 700.

I'm not sure what we were doing in 3.9 for certs and permissions here, but it doesn't work without this for me.